javaee8/jaxrs-2.1/src/main/java/org/wildfly/swarm/ts/javaee8/jaxrs/sse/SseClient.java

```diff
@@ -0,0 +1,38 @@
+package org.wildfly.swarm.ts.javaee8.jaxrs.sse;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.sse.SseEventSource;
+
+@Path("/client")
+public class SseClient {
+    @GET
+    @Path("/simple")
+    public String getSimpleEvent() {
+        return getClient("http://localhost:8080/simple");
+    }
+
+    @GET
+    @Path("/register")
+    public String getRegisteredEvent() {
+        return getClient("http://localhost:8080/broadcasting/register");
+    }
+
+    @GET
+    @Path("/broadcast")
+    public String getBroadcast() {
+        return getClient("http://localhost:8080/broadcasting/broadcast");
+    }
+
+    private static String getClient(String url) {
```

Comment:
This method doesn't return a "client", so `getClient` is a bad name. I'd just call it `get`.

Reply:
Or `getSse`, or something like that.